app/views/brexit_checker/results.html.erb

```diff
@@ -96,7 +96,20 @@
         } %>
       <% end %>
     <% else %>
-      <%= render 'stay_updated' %>
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+      <%= render 'components/email_link', {
+        data_attributes: {
+          "module": "track-click",
+          "track-action": action_based_email_link_label,
+          "track-category": "StayUpdated",
+          "track-label": transition_checker_email_signup_path
+        },
+        link_text: action_based_email_link_label,
```

Comment:
Note to self: this is consistent with the [other instance](https://github.com/alphagov/finder-frontend/pull/2354/commits/f0fc3568fa82e475ccf415918dc81779cb350bbb#diff-2ea979876c23d52fa27a5aa788b7a5dec0d25ff553e9751c11dd7d110627ca56R80) of the link.